platforms/hyperledger-indy/configuration/README.md

```diff
@@ -1,4 +1,45 @@
 # Playbooks
 
 ## Getting started
-This folder contains Ansible playbooks and their corresponding roles which are used to deploy Hyperledger Indy on Managed Kubernetes Cluster(s).
\ No newline at end of file
+This folder contains Ansible playbooks and their corresponding roles which are used to deploy Hyperledger Indy on Managed Kubernetes Cluster(s).
+
+
+## Prerequisites
+
+To run the playbooks, following are the pre-requisites.
+1. Ansible 2.8.2 with jmespath installed (a docker image can be made from this [Dockerfile](../../shared/images/ansibleSlave.Dockerfile)).
+2. Ansible controller configured like this [sample](../../shared/inventory) inventory settings.
+3. One Managed Kubernetes cluster for each organization (currently EKS tested so you need **AWS cli** credentials).
```

Comment:
This has to be corrected as Indy is only tested on vanilla Kubernetes version 1.16